MLS.tex

```diff
@@ -10,6 +10,10 @@
 % Add new Modelica Language logotype
 % The header ruler looks odd as Modelica Language define a natural line that is further up
 % We also need to fill the vertical space on the right
+% Not  using page numbers, since we use reference using sections.
```

Comment:
This sentence isn't quite right. Do you mean "since we use reference sections", or "we reference using sections" ?

Reply:
Hmm... "since we usually reference using sections."?

Reply:
Went with "Not using page numbers in right-header, since we usually refer to sections."